ceresdb-rpc/src/main/java/io/ceresdb/rpc/RpcOptions.java

```diff
@@ -91,6 +91,11 @@ public class RpcOptions implements Copiable<RpcOptions> {
 
     private boolean logOnLimitChange = true;
 
+    /**
+     * Max time in milliseconds a connection can live, 0 means forever.
+     */
+    private long connectionMaxAge = 0;
```

Comment:
```suggestion
    private long connectionMaxAgeMs = 0;
```
Maybe it would be better to add the time unit.